src/components/Home/summary.tsx

```diff
@@ -1,11 +1,7 @@
 import Statistics from '@/components/Home/statistics'
+import { Stat } from '@/types/types'
 
-const dummyVisitData = '12'
-const dummyDistanceData = '6.5'
-const dummyHourData = '3'
-const dummyDistanceCommuted = '20.5'
-
-function Summary() {
+function Summary(props: Stat) {
```

Comment:
You don't need to pass the stats in as props, you can access them via the hook (use the index page as an example).